File: discover_ruby/write_them_to_disc.rb
require 'HTTPClient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 39fed9cadb8579194e82b4bc4aefa34b8fab30b7'
}


clnt = HTTPClient.new
res = clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', query = nil, extheaders = {})

output = File.open("/tmp/36","w")
output << res
output.close
puts('The file was saved!')

File: discover_more_python/write_them_to_disc.py
import urllib2
import urllib

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 6be002d3b7968b5ef3fbbb318e97afe71225afe5'
}  

url = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'
req = urllib2.Request(url, None, request_headers)
response = urllib2.urlopen(req)
the_page = response.read()
# print (the_page)
file = open('/tmp/36', 'w+')
file.write(the_page)
file.close

print "The file was saved!"